.travis.yml

```
@@ -0,0 +1,12 @@
+notifications:
+  email: false
+
+language: csharp
+mono: none
+dotnet: 2.1
```

How about try using ?

```suggestion
dotnet: 2.2
sudo: required
dist: xenial
```

I had some problem with 2.2 but `dist: xenial` fixed it https://travis-ci.community/t/dotnet-core-2-2/1216/10

---

🤞

---

@shawnkoon ++